packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -547,9 +555,14 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
         );
     }
 
+    // NOTE: Search blob concept
+    public registerExtraSnapshotContracts(contracts: ISnapshotContracts) {
```

Comment:
I feel like it should be not plural. I.e. it's a contract, but it provides multiple blogs